### DISCUSSION TITLE
Help need Demo

### DISCUSSION THREAD

**Discussion by gMorning-Wp** [webpack/webpack#14948] [Q&A]
Can give me a copy of miniCssExtractPlugin and sass-loader combined with the webpack configuration as well as the corresponding version in package.json

**Comment by alexander-akait** [comment]
https://github.com/webpack-contrib/mini-css-extract-plugin#minimal-example, just put `sass-loader` after `css-loader`

**Reply by gMorning-Wp** [reply]
sorry,I didn't make it clear that I actually had some difficulty with sass and when using mini-css-extract-plugin, I didn't pull out the css, so I doubted the problem with the version in package.json, so my main concern was to have a corresponding version in package.json
